components/accelerator/nvidia/query/query.go

```diff
@@ -355,6 +383,26 @@ func (o *Output) GPUProductName() string {
 	return ""
 }
 
+func (o *Output) GPUProductNameFromSMI() string {
```

Comment:
From this page: https://developer.nvidia.com/management-library-nvml, my interpretation is nvidia-smi uses nvml so I think the ProductName and GPU count won't differ.

Reply:
Removed.